dces-drc-integration/src/integrationTest/resources/config.properties

```diff
@@ -0,0 +1,5 @@
+maat.api.base.url=${env:MAAT_API_BASE_URL}
+maat.api.oauth.base.url=${env:MAAT_API_OAUTH_BASE_URL}
+maat.api.oauth.client.id=${env:MAAT_API_OAUTH_CLIENT_ID}
+maat.api.oauth.client.secret=${env:MAAT_API_OAUTH_CLIENT_SECRET}
+maat.api.oauth.token.uri=${env:MAAT_API_OAUTH_URL}
```

Comment:
Isn't this duplicating data that could come from `integrationTest/resources/application.yaml`?

Reply:
Have folded this into the main application yaml. Good call!